res/lang/programme/programme_en.txt

```diff
@@ -106,46 +106,46 @@ PROGRAMME_PRODUCT_SHOW        = Show
 ONLY_X_BROADCASTS_POSSIBLE               = Only %X% broadcasts possible!
 NO_MORE_BROADCASTS_ALLOWED               = No more broadcasts allowed!
-BROADCAST_TIME_RESTRICTED                = Time of broadcast restricted
+BROADCAST_TIME_RESTRICTED                = Time of broadcast restricted!
 BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y       = Broadcast only allowed from %X%:00 to %Y%:00!
 FIRST_BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y = First broadcast only allowed from %X%:00 to %Y%:00!
 LIVE_PRODUCTION_FINISH_WILL_COST_X       = Live production finish will cost %X%
 
 OPENINGSHOW_TITLE       = Big launch of "%CHANNELNAME%"
-OPENINGSHOW_DESCRIPTION = Let's pop the corks. %CHANNELNAME% is launching today.
+OPENINGSHOW_DESCRIPTION = Let's pop the corks! %CHANNELNAME% is launching today.
 
 SPORT_PROGRAMME_TITLE                                = %SPORTNAME%: %LEAGUENAME% %SEASONYEARSTART%
 SPORT_PROGRAMME_ALL_X_MATCHES_OF_LEAGUEX_IN_SEASON_X = All %MATCHCOUNT% matches of %LEAGUENAME% in season %SEASONYEARSTART%.
 SPORT_PROGRAMME_MATCH_TIMES                          = Match times
-SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between place %TEAM1RANK% and %TEAM2RANK% of the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
-SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star %TEAM1STARSHORT% against the team around top scorer %TEAM2STAR%.
+SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between teams placed %TEAM1RANK% and %TEAM2RANK% in the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
+SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star, %TEAM1STARSHORT%, against top scorer %TEAM2STAR%'s team.
 SPORT_PROGRAMME_MATCH_DESCRIPTION3                   = An interesting game is coming up when %TEAM1ARTICLE1% %TEAM1NAME% meet the players of %TEAM2NAMEINITIALS%.
 SPORT_PROGRAMME_MATCH_DESCRIPTION4                   = %TEAM2ARTICLE1% %TEAM2NAME% visiting %TEAM1NAMEINITIALS% in %TEAM1CITY%.
-SPORT_PROGRAMME_MATCH_DESCRIPTION5                   = %TEAM2TRAINER% and his team travel to %TEAM2CITY%, rank %TEAM2RANK% in the league.
+SPORT_PROGRAMME_MATCH_DESCRIPTION5                   = %TEAM2TRAINER% and his team travel to %TEAM2CITY%, placed %TEAM2RANK% in the league.
 
-SPORT_PROGRAMME_PLAYOFF_MATCH    = Playoff match.
-SPORT_PROGRAMME_MATCH_OF_LEAGUEX = Match of %LEAGUENAME%.
-SPORT_PLAYOFFS_LONG              = "Playoffs"
-SPORT_PLAYOFFS_SHORT             = "PO"
+SPORT_PROGRAMME_PLAYOFF_MATCH    = Relegation match.
+SPORT_PROGRAMME_MATCH_OF_LEAGUEX = Match taking place in %LEAGUENAME%.
+SPORT_PLAYOFFS_LONG              = "Relegation"
+SPORT_PLAYOFFS_SHORT             = "Rel"
 
-SPORT_AFTER_PENALTY_SHORT  = "PSO"
+SPORT_AFTER_PENALTY_SHORT  = "Penalties"
 SPORT_AFTER_OVERTIME_SHORT = "AET"
 
-LIVE_ON_TAPE                        = Live on tape
+LIVE_ON_TAPE                        = Live recording
```

Comment:
For me "Live on tape" is a kind of technical term in broadcasting environments.
So .. what speaks against "live on tape" here?

Reply:
Just doesn't sound natural to me.  I've heard of a live recording but never a "live on tape", in fact that's a contradiction in terms.

Reply:
Live on Tape - means it is recorded on a tape without any "cuts" or additional "edits".

Reply:
Yeah, that's pretty much what I thought live recording meant.

Reply:
https://de.wikipedia.org/wiki/Live_on_tape
https://audio_video_en.en-academic.com/3252/live-to-tape
https://old.reddit.com/r/eurovision/comments/vbk8on/what_does_live_on_tape_mean/
https://www.quora.com/On-The-Colbert-Show-they-state-that-is-live-on-tape-Isn-t-that-an-oxymoron

...

Reply:
Interesting.  However, this site says "Formerly called live-on-tape, live recording includes all recording devices", implying that "live recording" is the newer term: https://quizlet.com/311116172/tv-production-chapter-17-flash-cards/

Reply:
I would prefer a more "common" link that just the quiz-page there. Something like a wiki entry or so at least.

Not all terms become "upgraded" after being technical "surpassed". Also speakers (the devices) are not just speaking :).

This aside: I am open to read the opinions of @scr0llbaer9000 and @nittka on this - was not intending to make a fuzz/big discussion out of it - so democratic decision at the end :)

Reply:
> https://de.wikipedia.org/wiki/Live_on_tape

It seems it's only in the German wikipedia though, so it might be a 'denglish' term, like "Handy" or "Mailbox". `;)`

At least originally. The next reference is Eurovision. By-and-large co-produced by German companies I guess. But as it then got mentioned in context of the Colbert show, maybe the term got some adoption in the English speaking world after all. But perhaps still not very popular.

Just my "guess" about the history of the term.

Reply:
I do not want your guess, I want your vote for either one of the proposed terms :-)

So - "live on tape" or "live record"?

Reply:
If "Live recording" sounds more natural, there is no reason not to use it in the localization.